contracts/Core/Parameters.sol

```diff
@@ -47,62 +49,66 @@ contract Parameters is IParameters, ACL {
         penaltyNotRevealDenom = _penaltyNotRevealDenom;
     }
-    
-    function setSlashPenaltyNum(uint256 _slashPenaltyNumerator) external onlyRole(DEFAULT_ADMIN_ROLE){
+
+    function setSlashPenaltyNum(uint256 _slashPenaltyNumerator) external onlyRole(DEFAULT_ADMIN_ROLE) {
         slashPenaltyNum = _slashPenaltyNumerator;
     }
-    
-    function setSlashPenaltyDenom(uint256 _slashPenaltyDenominator) external onlyRole(DEFAULT_ADMIN_ROLE){
+
+    function setSlashPenaltyDenom(uint256 _slashPenaltyDenominator) external onlyRole(DEFAULT_ADMIN_ROLE) {
         slashPenaltyDenom = _slashPenaltyDenominator;
     }
 
-    function setWithdrawLockPeriod(uint256 _withdrawLockPeriod) external onlyRole(DEFAULT_ADMIN_ROLE) { 
+    function setWithdrawLockPeriod(uint256 _withdrawLockPeriod) external onlyRole(DEFAULT_ADMIN_ROLE) {
         withdrawLockPeriod = _withdrawLockPeriod;
     }
 
     function setWithdrawReleasePeriod(uint256 _withdrawReleasePeriod) external onlyRole(DEFAULT_ADMIN_ROLE) {
-         withdrawReleasePeriod = _withdrawReleasePeriod;
+        withdrawReleasePeriod = _withdrawReleasePeriod;
     }
 
     function setResetLockPenalty(uint256 _resetLockPenalty) external onlyRole(DEFAULT_ADMIN_ROLE) {
         resetLockPenalty = _resetLockPenalty;
     }
-    
-    function setMaxAltBlocks(uint256 _maxAltBlocks) external onlyRole(DEFAULT_ADMIN_ROLE) { 
+
+    function setMaxAltBlocks(uint256 _maxAltBlocks) external onlyRole(DEFAULT_ADMIN_ROLE) {
         maxAltBlocks = _maxAltBlocks;
     }
 
-    function setEpochLength(uint256 _epochLength) external onlyRole(DEFAULT_ADMIN_ROLE) { 
+    function setEpochLength(uint256 _epochLength) external onlyRole(DEFAULT_ADMIN_ROLE) {
         epochLength = _epochLength;
     }
 
-    function setNumStates(uint256 _numStates) external onlyRole(DEFAULT_ADMIN_ROLE) { 
+    function setNumStates(uint256 _numStates) external onlyRole(DEFAULT_ADMIN_ROLE) {
         numStates = _numStates;
     }
 
-    function setExposureDenominator(uint256 _exposureDenominator) external onlyRole(DEFAULT_ADMIN_ROLE) { 
+    function setExposureDenominator(uint256 _exposureDenominator) external onlyRole(DEFAULT_ADMIN_ROLE) {
         exposureDenominator = _exposureDenominator;
     }
 
-    function setMinStake(uint256 _minStake) external onlyRole(DEFAULT_ADMIN_ROLE) { 
+    function setMinStake(uint256 _minStake) external onlyRole(DEFAULT_ADMIN_ROLE) {
         minStake = _minStake;
     }
 
-    function setGracePeriod(uint256 _gracePeriod) external onlyRole(DEFAULT_ADMIN_ROLE) { 
+    function setGracePeriod(uint256 _gracePeriod) external onlyRole(DEFAULT_ADMIN_ROLE) {
         gracePeriod = _gracePeriod;
     }
 
     function setAggregationRange(uint256 _aggregationRange) external onlyRole(DEFAULT_ADMIN_ROLE) {
         aggregationRange = _aggregationRange;
     }
 
+    function disableEscapeHatch() external onlyRole(DEFAULT_ADMIN_ROLE) {
+        escapeHatchEnabled = false;
```

Comment:
A toggle like feature can work here, right? Because once this is disabled, it can not be enabled. Or is this what we actually want to happen?

Reply:
Yes, this is expected imo

Reply:
It's what's supposed to happen. We will disable the escape hatch once the network is stable and without issues.

Reply:
Alright